DeepSpeech.ipynb

```diff
@@ -1435,7 +1404,7 @@
     "    time_started = datetime.datetime.utcnow()\n",
     "    \n",
     "    # Train the network\n",
-    "    last_train_wer, last_validation_wer = train(session, data_sets)\n",
+    "    last_train_wer, last_dev_wer = train(session, data_sets)\n",
```

Comment:
Why that? They (were also originally meant to) represent the values of the last epoch.